test/providers.js

```diff
@@ -1689,4 +1689,104 @@ describe('#linkedin', function () {
             });
         });
     });
+
+    describe('#phabricator', function () {
+
+        it('fails with no uri', { parallel: false }, function (done) {
+
+            var mock = new Mock.V2();
+            mock.start(function (provider) {
+
+                var server = new Hapi.Server();
+                server.connection({ host: 'localhost', port: 80 });
+                server.register(Bell, function (err) {
+
+                    expect(err).to.not.exist();
+
+                    expect( function() {
```

Comment:
you can rewrite as

`expect(Bell.providers.phabricator).to.throw(Error);`   <-- I think